.github/CODEOWNERS

```diff
@@ -153,8 +153,8 @@
 /tensorflow_addons/optimizers/yogi.py @manzilz
 /tensorflow_addons/optimizers/tests/yogi_test.py @manzilz
 
-/tensorflow_addons/rnn/cell.py @qlzh727 @pedrolarben
-/tensorflow_addons/rnn/tests/cell_test.py @qlzh727 @pedrolarben
+/tensorflow_addons/rnn/cell.py @qlzh727 @pedrolarben @failure-to-thrive
```

Comment:
I think we probably want to split the file in future, so that the PR/issue will be forward to the correct person.

Reply:
Yes, I think it would be better

Reply:
> I think we probably want to split the file in future, so that the PR/issue will be forward to the correct person.

@gabrieldemarmiesse, could you please take a look?

Reply:
I agree, not all cell classes should be in the same file. We can move the code of zoneoutLSTM in another file.

Reply:
To simplify the process, let's merge that PR and split the `cell.py` and `cell_test.py` thereafter. Those files contains lots of different cells already.